launch/launch/launcher.py

```diff
@@ -174,19 +173,21 @@ def _run(self):
         # terminate all remaining processes
         if all_futures:
 
-            # sending SIGINT to remaining processes
-            for index in all_futures.values():
-                p = self.task_descriptors[index]
-                if 'transport' in dir(p):
-                    self._process_message(p, 'signal SIGINT')
-                    try:
-                        p.transport.send_signal(signal.SIGINT)
-                    except ProcessLookupError:
-                        pass
+            # sending SIGINT to subprocess transport is not supported on Windows
```

Comment:
Should we link to the appropriate issue in comments?

https://github.com/python/asyncio/issues/191
https://groups.google.com/forum/#!topic/python-tulip/pr9fgX8Vh-A